pkg/apis/kubermatic/v1/project.go

```diff
@@ -69,6 +69,8 @@ type Project struct {
 type ProjectSpec struct {
 	// Name is the human-readable name given to the project.
 	Name string `json:"name"`
+	// AllowedOperatingSystems shows if the operating system is allowed to be use in the machinedeployment inside this project.
```

Comment:
AllowedOperatingSystems defines a map of operating systems that can be used for the machines inside this project.